docs/build/integrations/indexers/sentio.md

```diff
@@ -0,0 +1,269 @@
+
+### Fund Flows
+
+Sentio provides detailed and **ordered** fund flows. In the following example we visualize the process of how an arbitrageur made a profit by utilizing several trading venues.
+
+<Figure caption="Fund Flows" src={require('./img/sentio6.png').default } width="100%" />
+
+## Trace and Call
+
+Sentio provides trace view of transactions.
+
+### Trace modes and options
+
+**Trace mode:** Full trace mode includes cross-contract calls (CALL) and in-contract calls (JUMP).
+
+<Figure caption="Full Trace" src={require('./img/sentio7.png').default } width="100%" />
+
+You can also hide in-contract calls (JUMP) by turning off Full trace.
+
+**Options:** Users can hide static calls and select the level of trace displayed.
+
+<Figure caption="Trace Levels" src={require('./img/sentio8.png').default } width="100%" />
+
+**Call Graph:** Sentio provides the call graph that shows the contract interactions within a transaction.
+
+<Figure caption="Call Graph" src={require('./img/sentio9.png').default } width="100%" />
+
+## Debugging
+
+To understand a transaction even further, developers can use the **Debugger** tab to visualize the execution line-by-line.
+
+<Figure caption="Debugger" src={require('./img/sentio10.png').default } width="100%" />
+
+### Debugger tab layout
+
+**Traces**
+
+On the upper-left section, Sentio shows the trace of the transaction, this is the same as *trace and call.* Users can use this to select a location and execute directly to that position.
+
+<Figure caption="Call Trace Window" src={require('./img/sentio11.png').default } width="100%" />
+
+
+**Stack Traces**
+
+The bottom-left section contains the current call stack information, for example:
+
+<Figure caption="Stack Trace Window" src={require('./img/sentio12.png').default } width="100%" />
+
+### Single-Step Mode
+
+:::info
+To use single-step mode:
+- Turn on single-step mode.
+- (optional) Use Debug Build -- Sentio will recompile the contract with different compiler parameters to achieve the best source mapping. See **Limitations** below.
+:::
+
+<Figure caption="Debugger Options" src={require('./img/sentio13.png').default } width="100%" />
+
+
+**Use Debugger**
+
+The debugger has standard definitions of:
+
+- Step-Over: Move to the next line of execution.
+- Step-Into: If there is a function, steps into the function.
+- Step-Out: If we are in a function, steps out the function to the upper level.
+- Continue: This is the standard break-point.
+- Restart: Restart from the beginning.
+
+**Inspect Variables**
+
+The debugger automatically shows the local variables within the call context, and all the contract variables.
+
+<Figure src={require('./img/sentio14.png').default } width="100%" />
+
+The debugger also supports adding **user defined watched variables (similar to a regular debugger.)**
+
+<Figure src={require('./img/sentio15.png').default } width="100%" />
+
+**Limitations**
+
+- Contracts compiled with viaIR option are not fully supported.
```

Comment:
Is `viaIR` a special word, or is it missing a space?

Reply:
viaIR enables the new Yul-based compiler pipeline for Solidity